src/config/templates.toml

```diff
@@ -47,6 +47,8 @@ show = 'show'
 
 # Hook points for users to customize the default templates:
 'format_short_id(id)' = 'id.shortest(12)'
+'format_change_id(id)' = 'format_short_id(id)'
+'format_commit_id(id)' = 'format_short_id(id)'
```

Comment:
nit: `format_short_commit_id()`/`_change_id()` because `jj show` doesn't use these for formatting full id.